[Installation] Installation not fully documented

## Expected Behavior
I should be able to follow the instructions from the [web site](https://foundation.zurb.com/sites/docs/installation.html) (using Foundation-CLI) and those in readme.md in the zurb full site template, pretty much word-for-word from and end up with a working site.

## Current Behavior
- Running `foundation new` causes errors if python is not installed
- After installing python using npm, I followed the instructions in readme.md in the template and running `yarn` causes 'yarn' is not recognized as an internal or external command

## Possible Solution
If someone tried this on a clean machine, they would know what elements are missing and update the documentation to include pre-requisites with links.

## Test Case and/or Steps to Reproduce (for bugs)
How to reproduce:
1. Clean build of Windows 10 (mine is not a clean build but is still missing the pre-reqs)
2. Run `foundation new` and observe errors due to missing python install
3. After installing windows-build-tools using npm, run yarn in the root directory as described in readme.md and observe that yarn is not a known command

## Context
My first attempt to create a blank foundation site to evaluate it as a framework

## Your Environment
- Foundation version(s) used: 6.4.3
- Browser(s) name and version(s): N/A
- Operating System and version (desktop or mobile): Windows 10 1803 desktop
- Link to your project: N/A

## Checklist (all required):
- [x] I have read and follow the [CONTRIBUTING](CONTRIBUTING.md) document.
- [x] This is a bug report or a feature request.
- [x] There are no other issues similar to this one.
- [x] The issue title is descriptive.
- [x] The template is fully and correctly filled.

I tried the commands above and here are the results:

1. Run `foundation new` - WORKED
2. Changed "package.json" from "~6.4.1" to "~6.4.3" in the "dependencies" section - WORKED
3. Run `npm ci` - ERROR

npm WARN prepare removing existing node_modules/ before installation
npm ERR! cipm can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.
npm ERR!
npm ERR!
npm ERR! Missing: gulp@gulpjs/gulp#4.0
npm ERR!

4. Run `foundation watch` - FAILED

I ran `npm install` based on the suggestion in the error message in #3 above and installed everything. Then I ran `foundation watch` and everything worked.

I still ended up with a directory that is 330MB instead of 17MB as mentioned above, but I guess there is no way around that if you use `npm install` to get everything set up instead of just `foundation new`.

--- a/docs/pages/installation.md
+++ b/docs/pages/installation.md
@@ -185,6 +185,17 @@ After you selected "Foundation for Sites", Foundation CLI will ask you which tem
   <p>To remove the Foundation 5 CLI, run <code>gem uninstall foundation</code>. After testing the Foundation 6 CLI, if you want to remove it to go back to the old CLI, run <code>npm uninstall foundation-cli --global</code>.</p>
 </div>
 
+<div class="callout info">
+  <p><strong>Windows users</strong>: make sure you've python v2.7 available in your node environment since it's required by the node-gyp tool. There are two way to achieve this</p>
+
+  <ol>
+    <li>Install the <a href="https://github.com/felixrieseberg/windows-build-tools">windows-build-tools</a> (recommended) and make python afterwards accessible via <code>npm config set python "%USERPROFILE%\.windows-build-tools\python27\python.exe"</code></li>
+    <li>Install <a href="https://www.python.org/downloads/">python</a> (not recommended) and add it to your system environment variables</li>
+  </ol>
+ 
+  <p>The first way is recommended if you've not installed pythong v2.7 yet as it doesn't affect your machine outside the node environment. In case you've already installed python v2.7 you may of course skip both ways and start using foundationc-cli immediately.</p>
+</div>
+
 ---
 
 ## Download

